src/connectors/Venly/index.ts

```diff
@@ -0,0 +1,135 @@
+import { ExternalProvider, Web3Provider } from '@ethersproject/providers';
+import { NoEthereumProviderError } from '@pangolindex/web3-react-injected-connector';
+import { Venly } from '@venly/web3-provider';
+import { AbstractConnector } from '@web3-react/abstract-connector';
+import { AbstractConnectorArguments, ConnectorUpdate } from '@web3-react/types';
+import warning from 'tiny-warning';
+import { UserRejectedRequestError } from '../DefiConnector';
+
+const venlyClientID = process.env.VENLY_ID ?? 'Testaccount';
```

Comment:
do we have VENLY_ID, so that I can set it into github action secrets

Reply:
Not yet, i asked abdullah to fill out the Venly form so we can get this id